shared/src/commonMain/kotlin/world/respect/domain/opds/model/OpdsFeedMetadata.kt

```diff
@@ -0,0 +1,35 @@
+package world.respect.domain.opds.model
+
+import kotlinx.serialization.Contextual
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import java.time.LocalDateTime
+
+/**
+ * Represents metadata for an OPDS catalog or collection.
+ * This includes details like title, number of items, items per page, etc.
+ *
+ * For reference, see the schema: https://github.com/readium/webpub-manifest/blob/master/schema/metadata.schema.json
```

Comment:
Wrong reference link for schema: for feed should be https://drafts.opds.io/schema/feed-metadata.schema.json